### ISSUE TITLE
eth_getTransactionReceipt returning null for certain transaction hashes

### ISSUE THREAD

**Issue by aaron-cottrell-vc** [hashgraph/hedera-json-rpc-relay#2559]
### Description

One of our customers has reported issues on testnet with transaction hashes that were returned from the export blocks and transaction methods failing when querying them using `eth_getTransactionReceipt` RPC method.

Some follow-up discussion with the Swirlds team included mention of a potential known issue of synthetic transactions being listed in a block's transactions which may be a cause.

### Steps to reproduce

On testnet the `eth_getTransactionReceipt` returns null for certain transaction hashes.

Example transaction hashes:
```
0xb39e77ab273c7f15421e08357a94d3763493e04ff6dc15343412256066b6a26a
0x86cc47921f479b7bbd0faa58a7a9123becdad319e7fc8ab2e5a848ee823229e4
0x5a53839382af994dd19ab16290214d9444fe493bdf5bea0bdcc26fb9c37c9518
```

We have confirmed this on our node and also the hashio public endpoint. 

We also couldn't find these transaction hashes using an explorer, e.g. https://hashscan.io/testnet/search-result/0xb39e77ab273c7f15421e08357a94d3763493e04ff6dc15343412256066b6a26a?errorCount=0

**Example request:**
```
curl --request POST \
  --url https://testnet.hashio.io/api \
  --header 'Content-Type: application/json' \
  --data '{
	"id": 1,
	"jsonrpc": "2.0",
	"method": "eth_getTransactionReceipt",
	"params": [
		"0xb39e77ab273c7f15421e08357a94d3763493e04ff6dc15343412256066b6a26a"
	]
}'
```

**Example response:**
```
{
	"jsonrpc": "2.0",
	"result": null,
	"id": 1
}
```

**Expected result:**
Return the transaction receipt data for the given transaction hash.

### Hedera network

testnet

### Version

v0.48.0

### Operating system

None

**Comment by Nana-EC** [comment]
Thanks for the issue @aaron-cottrell-vc 
By any chance due you have any of the blocks identifiers that the customers believe to contain the said transaction?
A block to transaction hash pair would assist investigation of those specific instances.

**Comment by ebadiere** [comment]
@aaron-cottrell-vc Yes, these are synthetic transactions.

**Comment by ebadiere** [comment]
@aaron-cottrell-vc We're exploring options.  We'll get back to you.

**Comment by Nana-EC** [comment]
@aaron-cottrell-vc latest issue to track is https://github.com/hashgraph/hedera-mirror-node/issues/8759
We have a direction but it requires a HIP and time to implement. The team is working on it. Thanks

**Comment by ebadiere** [comment]
Resolved in release `v0.53.0`

```
 curl --request POST \
  --url https://testnet.hashio.io/api \
  --header 'Content-Type: application/json' \
  --data '{
        "id": 1,
        "jsonrpc": "2.0",
        "method": "eth_getTransactionReceipt",
        "params": [
                "0xb39e77ab273c7f15421e08357a94d3763493e04ff6dc15343412256066b6a26a"
        ]
}'
{"result":{"blockHash":"0x590b9697a9b81711149e6fe768d6667ad4bf8111a9ab4f4738b458f32bc307a9","blockNumber":"0x479b2f","contractAddress":"0x00000000000000000000000000000000003cbc98","cumulativeGasUsed":"0x0","effectiveGasPrice":"0xbeebcf0800","from":"0x0000000000000000000000000000000000000000","gasUsed":"0x0","logs":[{"address":"0x00000000000000000000000000000000003cbc98","blockHash":"0x590b9697a9b81711149e6fe768d6667ad4bf8111a9ab4f4738b458f32bc307a9","blockNumber":"0x479b2f","data":"0x000000000000000000000000000000000000000000000000000000000002ba86","logIndex":"0x0","removed":false,"topics":["0x8c5be1e5ebec7d5bd14f71427d1e84f3dd0314c0f7b2291e5b200ac8c7c3b925","0x000000000000000000000000000000000000000000000000000000000028a4a2","0x00000000000000000000000000000000000000000000000000000000003cbca5"],"transactionHash":"0xb39e77ab273c7f15421e08357a94d3763493e04ff6dc15343412256066b6a26a","transactionIndex":"0x1"}],"logsBloom":"0x00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000","root":"0x0000000000000000000000000000000000000000000000000000000000000000","status":"0x1","to":"0x00000000000000000000000000000000003cbc98","transactionHash":"0xb39e77ab273c7f15421e08357a94d3763493e04ff6dc15343412256066b6a26a","transactionIndex":"0x1","type":null},"jsonrpc":"2.0","id":1}%
```
